scripts/india_wris/India_WRIS_Ground/India_WRIS_Ground.tmcf

```diff
@@ -1,232 +1,193 @@
 Node: E:India_WRIS_Ground->E0
 dcid: C:India_WRIS_Ground->dcid
 typeOf: dcs:WaterQualitySite
+#containedIn: E:India_WRIS_Ground->E0
```

Comment:
Could you add a comment on why these are commented out? ContainedIn for the measurement sites seems reasonable (unless there was a resolution error)?

Reply:
yes, the containedIn is moved to a separate Geo import for water quality stations.

Reply:
got it. in that case, could we delete it here, so that there aren't merge conflicts later?